community/kernel/src/main/java/org/neo4j/kernel/impl/factory/CommunityEditionModule.java

```diff
@@ -99,6 +100,8 @@ public CommunityEditionModule( PlatformModule platformModule )
 
         GraphDatabaseFacade graphDatabaseFacade = platformModule.graphDatabaseFacade;
 
+        dependencies.satisfyDependency( SslPolicyLoader.create( config, logging.getInternalLogProvider() ) ); // for bolt and web server
```

Comment:
Why this is for bolt and web server if SslPolicyLoader rejects Legacy ssl settings? Can we have some tests to verify the old behaviour works as before?